test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
delete file as we do not use fixtures. might be a good idea to remove rails generator from gnerating this file in the future